examples/website/scenegraph-layer/style/app.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
Is this file used?

Reply:
Copied the whole file from another example. Not sure. Let me try to delete it and see what happens.

Reply:
Not needed, I will remove.